src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicSyntaxFactsService.vb

```diff
@@ -114,12 +139,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
         Public Function IsObjectCreationExpressionType(node As SyntaxNode) As Boolean Implements ISyntaxFactsService.IsObjectCreationExpressionType
             Return node.IsParentKind(SyntaxKind.ObjectCreationExpression) AndAlso
-                DirectCast(node.Parent, ObjectCreationExpressionSyntax).Type Is node
+                    DirectCast(node.Parent, ObjectCreationExpressionSyntax).Type Is node
```

Comment:
Looks like indentation got massively horked in this file.  Can you jsut get the original source and paste that in?